services/ui-src/src/components/pages/Admin/AdminPage.tsx

```diff
@@ -63,11 +59,11 @@ export const AdminPage = () => {
                     </Text>
                     <Text sx={sx.currentBannerDate}>
                       Start Date:{" "}
-                      <span>{convertDateUtcToEt(bannerData?.startDate)}</span>
+                      <span>{convertDateUtcToEt(bannerData?.startDate!)}</span>
```

Comment:
I think this needs a change, but I'm not sure what it should be.

For one, the `?` in `bannerData?.startDate` was never necessary here; we know that `bannerData` exists from the check on line 51.

More importantly, the `!` worries me. We're telling Typescript "trust me, this value will always exist"... but you and I know that it _won't_ always exist, because the field in AdminBannerForm is optional. What will we display for a banner with no defined start date? Today's date? An error? Nothing?

I haven't tested this, but I suspect (based on my reading of `convertDateUtcToEt`) that it will be today's date, which would be incorrect behavior, right? Do you think the best behavior would be to display nothing instead? Something like: `{ bannerData.startDate && <span{convertDate...` perhaps?

If you've already tested this scenario and are satisfied with the behavior, my apologies - but maybe we should make that behavior explicit? Something like: `bannerData.startDate ?? new Date().valueOf()` perhaps? Or maybe a change within `convertDateUtcToEt`, so that it accepts `number | undefined`?

And whatever change we make here, we should also make 4 lines down for `endDate`.

Reply:
I'm glad you gave feedback on this because I was thinking about this before starting work today and wasn't certain what to do. 

The start & end date field isn't suppose to be optional when filling out the admin banner form as there's a set time frame when the banner would run, allowing for scheduling ahead of time. I think this is why it was split as a separate type that inherited from a banner parent type. There's suppose to be a validation check that would make sure the user enter's the date fields in but we currently don't have validation set up yet.

Anyway, I decided to go with your suggestion of the conditional to check if the value exist and to display nothing if somehow the user managed to save without it.